public/cli/lib/rendering/recordFrames.ts

```diff
@@ -12,12 +12,10 @@ export async function record(
   logEachFrame = false,
   notifyEvent?: EventEmitter
 ) {
-  // chromium.path may or may provide a path in an asar archive.  If it does
-  // it is unusable, and we'll attempt to swap it out for the un-archived version
-  const chromiumPath = chromium.path.replace('app.asar', 'app.asar.unpacked');
-
   const browser = await puppeteer.launch({
-    executablePath: chromiumPath,
+    // chromium.path may or may provide a path in an asar archive.  If it does
```

Comment:
nit: please fix the comment by adding 'not' to read 'chromium.path may or may not provide...'

Reply:
Thank you.